scripts/scan-pdf-content.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+"""
+Scan user uploaded PDFs in S3 and flag ones which contain any
+non-text or image content (ie. JS or videos).
+
+Requires a running version of veraPDF-REST (https://github.com/veraPDF/veraPDF-rest)
+
+Set AWS_PROFILE to the relevant profile for the stage when running eg.
+$ AWS_PROFILE=development-developer ./scripts/scan-pdf-content.py preview g-cloud-12 http://localhost:8080
+
+Usage: scan-pdf-content.py <stage> <framework> <verapdf-url> [--index=<index>]
+
+Options:
+    <stage>                         Stage to target
+    <framework>                     Slug for the framework to scan
+    <verapdf-url>                   The url for the verapdf-rest service to use
+    --index=<index>                 Optional. If supplied, the scan resumes from a previous run, using
+                                        the list of PDFs stored in '/tmp/pdf-scan-queue.json' starting at `index`
+
+    -h, --help                      Show this information
+
+Examples:
+    ./scripts/scan-pdf-content.py staging g-cloud-12 http://localhost:8080
+"""
+import boto3
+import requests
+import csv
+import logging
+import json
+import os
+import threading
+import queue
+from docopt import docopt
+from datetime import datetime
+
+from typing import Tuple, NamedTuple, Optional
```

Comment:
It's considered [good style](https://google.github.io/styleguide/pyguide.html#313-imports-formatting) to group imports by type, so standard library imports first, then third-party libraries, then our own libraries last. (I realise we don't follow this rule everywhere in our own code, so sorry for picking you up on it!).

```suggestion
import csv
import json
import logging
import os
import threading
import queue
from datetime import datetime
from typing import Tuple, NamedTuple, Optional

import boto3
import requests
from docopt import docopt
```